ADR.md

```diff
@@ -0,0 +1,33 @@
+# Architecture Decision Record
+
+Architecture Decision Records are a lightweight way of keeping track of critical design decisions and the reasoning behind them.
+
+For further information, please checkout <https://adr.github.io> or [https://heise.de (German)](https://www.heise.de/hintergrund/Gut-dokumentiert-Architecture-Decision-Records-4664988.html?seite=all).
+.template
+## Placement
+
+It's recommended to create a folder `architecture` in the relevant crate (if the change is scoped to that trait) or the manifest root, where all project overarching ADRs are collected.
+
+## Template
+
+We are useing a derivation of a public [template by Michael Nygard](https://github.com/joelparkerhenderson/architecture-decision-record/blob/main/templates/decision-record-template-by-michael-nygard/index.md):
+
+```md
+# Title
+
+## Status
+
+<!-- What is the status, such as proposed, accepted, rejected, deprecated, superseded, etc.? -->
+
+## Context
+
+<!-- What is the issue that we're seeing that is motivating this decision or change? -->
+
+## Decision
+
+<!-- What is the change that we're proposing and/or doing? -->
+
+## Consequences
+
+<!-- What becomes easier or more difficult to do because of this change? -->
+```
```

Comment:
Is this something that was discussed somewhere already?

Reply:
This sounds like quite a significant change for the existing workflow. Thus I would expect a discussion to happen regarding it. At the same time, I have missed quite some of the parachains calls. So I thought maybe I could catch up with that or get a gist of that discussion somewhere.

Reply:
This repo is also not just about parachains. I don't have seen any wider discussion about this change somewhere.

Reply:
The intent was mostly to guide people, and the ADR is something that's an approach to document things and reduce the tribal knowledge on why things are as they are.